### PR TITLE
LPS-115915 Show drafts of the users.

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:forward

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/search/ContentDashboardIndexerPostProcessor.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import com.liferay.portal.kernel.util.GetterUtil;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Gregory Amerson
```
Comment (darquesdev) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/search/ContentDashboardIndexerPostProcessor.java:35]: ouch!

Comment (cgoncas) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/search/ContentDashboardIndexerPostProcessor.java:35]: hahaha

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**